src/TurbulenceClosures/turbulence_closure_implementations/constant_anisotropic_diffusivity.jl

```diff
@@ -39,28 +39,20 @@ end
 
 calculate_diffusivities!(K, arch, grid, closure::ConstantAnisotropicDiffusivity, args...) = nothing
 
-@inline ∂ⱼ_2ν_Σ₁ⱼ(i, j, k, grid, closure::ConstantAnisotropicDiffusivity, U, args...) = (
-      closure.νh * ∇h²_fca(i, j, k, grid, U.u)
-    + closure.νv * ∂z²_aac(i, j, k, grid, U.u)
-    )
+@inline ∂ⱼ_2ν_Σ₁ⱼ(i, j, k, grid, closure::ConstantAnisotropicDiffusivity, U, args...) =
+    div_ν∇u(i, j, k, grid, closure.νh, closure.νh, closure.νv, U.u)
 
-@inline ∂ⱼ_2ν_Σ₂ⱼ(i, j, k, grid, closure::ConstantAnisotropicDiffusivity, U, args...) = (
-      closure.νh * ∇h²_cfa(i, j, k, grid, U.v)
-    + closure.νv * ∂z²_aac(i, j, k, grid, U.v)
-    )
+@inline ∂ⱼ_2ν_Σ₂ⱼ(i, j, k, grid, closure::ConstantAnisotropicDiffusivity, U, args...) =
+    div_ν∇v(i, j, k, grid, closure.νh, closure.νh, closure.νv, U.v)
 
-@inline ∂ⱼ_2ν_Σ₃ⱼ(i, j, k, grid, closure::ConstantAnisotropicDiffusivity, U, args...) = (
-      closure.νh * ∇h²_cca(i, j, k, grid, U.w)
-    + closure.νv * ∂z²_aaf(i, j, k, grid, U.w)
-    )
+@inline ∂ⱼ_2ν_Σ₃ⱼ(i, j, k, grid, closure::ConstantAnisotropicDiffusivity, U, args...) =
+    div_ν∇w(i, j, k, grid, closure.νh, closure.νh, closure.νv, U.w)
```

Comment:
I'm a bit confused by this notation. We have a tensor diffusivity here, which means we cannot write this operator as "the divergence of `ν∇w`". Why is this name for the operator preferred, when it does not correctly reflect the mathematical operation? Is there some kind of benefit to this shorthand? I feel the explicit original versions are easier to read and understand.

Reply:
That's a good point. Originally I just wanted to get rid of things like `∇_κ∇c` as it looked like a "gradient of a gradient". Indeed `div_ν∇w` is quite specific so I'll change them to use names like `∂ⱼνᵢⱼ∂ᵢw`.